packages/node_modules/nav-frontend-modal/src/index.tsx

```diff
@@ -81,9 +81,7 @@ class ModalWrapper extends React.Component<ModalProps, {}> {
         const { onRequestClose, shouldCloseOnOverlayClick } = this.props;
         if (shouldCloseOnOverlayClick || evt.type === 'keydown') {
             onRequestClose();
-        } else if (this.closeButtonRef) {
-            this.closeButtonRef.focus();
-        } else {
+        }   else {
```

Comment:
Burde ikke `else if` biten også være med? 
Evt om man ikke skal sette fokus på knappen så kan hele `closeButtonRef` fjernes for da tror jeg ikke den er i bruk noe mer.

Reply:
Hm, denne er nok min skyld. Er det ikke litt rart å sette slik fokus inne i `onRequestClose`? 🤔

Reply:
Cases om blir dekt er om man har satt `shouldCloseOnOverlayClick=false` og deretter klikker på utsiden av modal, fokus ble da satt til lukk-knappen. Om det fjernes så vil det istedetfor settes til modalen i sin helhet.
Ser at all fokus-funksjonaliteten ble lagt til samtidig som `shouldCloseOnOverlayClick` ble lagt til for litt over tre år siden.  

Men hvorvidt vi vil ha den funksjonaliteten er jo noe man kan ta en runde på. Men tenker det passer bedre i en egen PR da det er en breaking change og vil kreve en major-bump

Reply:
Aha, ja da bør nok denne bli med videre enn så lenge 👍

Reply:
Problemet her er at Xknapp ikke har noe `focus property` slik Lukknapp hadde. Fikset ved å lage en chain av ref og focus fra `Modal -> Xknapp -> KnappBase`. Er det noen bedre måte å løse dette på? Burde kanskje `focus` implementeres på alle knapper i systemet?

Reply:
Den korrekte måten å håndtere dette på er vel egentlig [RefForwarding](https://reactjs.org/docs/forwarding-refs.html). Det har vi også ett issue på som er lagt til V-next-milestonen #587. 

Men ikke sikkert det er hensiktsmessig å gjøre de nødvendige endringene i `Xknapp` og `KnappBase` akkurat nå? :/

Reply:
Enig, tenker det er best å vente med å deprecate `Lukknapp` da siden det er en del som må fikses først. Reverserer også alle endringer i Modal for å fortsette bruk av `Lukknapp`.